types/mdx/mdx-tests.tsx

```diff
@@ -47,6 +47,18 @@ declare function jsx(): TestElementType;
 // A JSX test implementation type
 declare global {
     namespace JSX {
+        // The addition and changes in ElementType change what is valid JSX.
+        // This should also be reflected by the MDX types.
+        // type ElementType =
+        //     | 'a'
+        //     | 'div'
+        //     | 'h1'
+        //     | 'img'
+        //     | 'span'
+        //     | 'video'
+        //     | ((props: Record<string, any>) => Element | null)
+        //     | (new (props: Record<string, any>) => ElementClass);
```

Comment:
It should be tested both with this included, and variations of this type. DefinitelyTyped can’t do that. In a real world application, `JSX.ElementType` may or may not have been defined.

We could require `ElementType` to be defined, but that’s a breaking change.

> Wouldn’t this commented-out code make more sense on L13, where `TestElementType` is defined?

No, that’s outside of the JSX namespace.

Reply:
> It should be tested both with this included, and variations of this type. DefinitelyTyped can’t do that. In a real world application, JSX.ElementType may or may not have been defined.

Maybe you can add that in a comment?

And what do you think on the note I added in my previous comment?

Reply:
I’m fine changing the comment. I’ll await the answer on https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65703#discussion_r1255771468, because if the we’re not allowed to use `@ts-ignore`, we’ll have to make `ElementType` required, which will be a breaking change.